workflow_imaging/ingest.py

```diff
@@ -1,8 +1,9 @@
 import pathlib
 import csv
+from datetime import datetime
 
-from .pipeline import subject, imaging, scan, Session, Equipment
-from .paths import get_imaging_root_data_dir
+from workflow_imaging.pipeline import subject, imaging, scan, Session, Equipment
+from workflow_imaging.paths import get_imaging_root_data_dir
```

Comment:
should these stay as relative imports?

Reply:
Cannot run as a script (`python workflow_imaging/ingest.py`) if those are relative imports